File: PKI-android/app/src/main/java/com/example/pki/model/Event.java
package com.example.pki.model;

import androidx.annotation.Nullable;
import java.util.Date;
import java.util.List;

public class Event {
    @Nullable
    public Integer eventId;
    public int numberOfPeople;
    public Date date;
    @Nullable
    public String comment;
    @Nullable
    public Integer grade;
    @Nullable
    public List<Notification> notificationList;
    public User customerId;
    public EventOffer eventOfferId;
    public EventStatus eventStatusId;
    public int hoverGrade = 0;
}

File: PKI-web/app/frontend/src/app/models/Event.ts
import { EventOffer } from "./EventOffer";
import { EventStatus } from "./EventStatus";
import { User } from "./User";

export class Event {
  eventId?: number;
  numberOfPeople!: number;
  date!: Date;
  comment?: string;
  grade?: number;
  notificationList?: Notification[];
  customerId!: User;
  eventOfferId!: EventOffer;
  eventStatusId!: EventStatus;
  hoverGrade: number = 0;

  constructor(init?: Partial<Event>) {
    Object.assign(this, init);
  }
}
